region/crateria/central/Landing Site.json

```diff
@@ -1252,6 +1313,40 @@
       "clearsObstacles": ["A"],
       "note": "Shinespark diagonally from the small hill left of the ship to break part way into the hidden bomb wall, then use Bombs or another means to continue to the left."
     },
+    {
+      "link": [5, 1],
+      "name": "Speedy Jump Diagonal Spark",
+      "requires": [
+        "h_canShineChargeMaxRunway",
+        "canShinechargeMovement",
+        "canMidairShinespark",
+        {"or": [
+          {"shinespark": {"frames": 29, "excessFrames": 1}},
+          {"and": [
+            "canTrickyJump",
```

Comment:
```suggestion
            "canShinechargeMovementComplex",
```

Reply:
I got 24 by running across the ship.
24 from jumping over the ship.
20 by jumping straight up left of ship.
It's hard to know where you should jump from to save energy.